xformers/sparse/_csr_ops.py

```diff
@@ -148,8 +149,9 @@ def backward(ctx, grad):
 
         # gradients w.r.t. values
         grad = grad.contiguous()
+        b = b.contiguous()
```

Comment:
this is fine, but note that this is not necessary now because we cast it as contiguous in forward